travis_stages/deploy.sh

```diff
@@ -1,8 +1,10 @@
 docker build -t freakency/devops_minitwit_app app/
 docker build -t freakency/devops_minitwit_api api/
+docker build -t freakency/devops_minitwit_stalker stalker/
 echo "$docker_password" | docker login -u "$docker_user" --password-stdin
 docker push freakency/devops_minitwit_app
 docker push freakency/devops_minitwit_api
+docker push freakency/devops_minitwit_stalker
```

Comment:
You need to tell me when you want to push to a new repository haha, I need to create it on my dockerhub 👍

Reply:
Hopefully it will work right away. Sometimes when you create a new repo you need to push your first image when the repo is in private and then put it back in public